Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewController.swift

```diff
@@ -6,9 +6,10 @@
 //  Copyright © 2021 Владислав Лисянский. All rights reserved.
 //
 
+import SharedRouter
 import UIKit
 
-public final class AboutScreenViewController: UIViewController {
+public final class AboutScreenViewController: UIViewController, Navigates {
```

Comment:
What it is used for in ViewController?

Reply:
No, it is was my mistake as it seems that I forgot to remove it.